README.md

```diff
@@ -44,6 +44,45 @@ Integration with JupyterLab [(Take me to the installation guide!)](https://githu
 
 A PyCharm integration is also in our roadmap so stay tuned!
 
+## 💨 Guide to the Roadmap
+
+### **What is coming up soon?**
+
+After gathering valuable feedback and comments from data scientists, we created the following short-term roadmap for genv:
+
+**1. Environment specific information:** With this feature, you will be able to see the processes, machines and utilization of the machines specifically to your GPU environment when you type nvidia-smi
+
+**2. Docker support:** Running Docker containers powered by the GPUs in your environment
+
+**3. Configuring GPU memory:** Specifying the GPU memory that needs to be configured in an environment
+
+**4. To be able to provisioning the GPU fractions**
```

Comment:
I think it's more correct to say: "To be able to provision fractions of GPUs"
wdyt?

Reply:
hahahaha yes you are right